Added ability to print (and save as pdf)

jamie-chiu opened the pull request:
As a simple solution for now, have added the ability to print the consent form (which also allows participant to save as PDF). This way, participants can always have the latest up-to-date version printed/saved; and by-passes the need for experimenters to upload a PDF.
Also added in slight padding to page.

jamie-chiu committed:
Added ability to print (and save as pdf)

szorowi1 commented:
@jamie-chiu, the print button looks and works great! 

Before I accept the changes, can I ask why the change to the CSS? (i.e. padding, line-height, etc). Is the consent form formatted strangely on your monitor? (I ask because I work mostly off one laptop, so I'm not always great at catching resolution issues on other monitors)

szorowi1 committed:
issue #108 / pr #109
- @szorowi1 suggested changes: set max-width in global CSS (tacit-css) 
so that text does not run flush with screen edge on large monitors

szorowi1 committed:
issue #108 / pr #109
- remove vestigial <div>

szorowi1 reviewed (approved):
Ok, everything looks good! Consider #108 closed. Thanks so much, @jamie-chiu!

At your earliest convenience, if you wouldn't mind adding the same changes to the MTurk branch, that would be a huge help.
